tests/helpers/pageObjects/footer.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from '@playwright/test';
+import type { Page } from '@playwright/test';
+
+export class Footer {
+  private development = this.page.locator('[data-test-id="development"]');
+  private features = this.page.locator('[data-test-id="features"]');
+  private ecosystem = this.page.locator('[data-test-id="ecosystem"]');
+  private education = this.page.locator('[data-test-id="education"]');
+  private community = this.page.locator('[data-test-id="community"]');
+  private contact = this.page.locator('[data-test-id="contact"]');
+  private whitepaper = this.page.locator('[data-test-id="whitepaper"]');
+  private roadmap = this.page.locator('[data-test-id="roadmap"]');
+  private faq = this.page.locator('[data-test-id="faq"]');
+
+  constructor(private page: Page) {}
+
+  async developmentUrl() {
+    await this.development.isVisible();
+    await this.development.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/btcv-explained');
+  }
+
+  async featuresUrl() {
+    await this.features.isVisible();
+    await this.features.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/features');
+  }
+
+  async ecosystemUrl() {
+    await this.ecosystem.isVisible();
+    await this.ecosystem.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/ecosystem');
+  }
+
+  async educationUrl() {
+    await this.education.isVisible();
+    await this.education.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/tutorials');
+  }
+
+  async communityUrl() {
+    await this.community.isVisible();
+    await this.community.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/community');
+  }
+
+  async contactUrl() {
+    await this.contact.isVisible();
+    await this.contact.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/contact');
+  }
+
+  async whitepaperUrl() {
+    await this.whitepaper.isVisible();
+    await this.whitepaper.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/whitepaper');
+  }
+
+  async roadmapUrl() {
+    await this.roadmap.isVisible();
+    await this.roadmap.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/roadmap');
+  }
+
+  async faqUrl() {
+    await this.faq.isVisible();
+    await this.faq.click();
+    await expect(this.page.url()).toMatch('https://bitcoinvault.global/faq');
```

Comment:
shouldn't this URL reflect the environment we are currently on? On Stage/Test env this will test if we are redirected from test to prod